Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,2482 @@
+define([
```

Comment:
Looks like it does more than it needs to given the limitations classification has.  Are you sure there aren't other limitations we can add to make this even more simple, e.g., vertex formats, removing the cache (is it needed), etc.

Then perhaps factor out some common functions to share with `Model`?